server/fleet/windows_mdm.go

```diff
@@ -68,8 +68,7 @@ func (m *MDMWindowsConfigProfile) ValidateUserProvided() error {
 			return errors.New("Only <Replace> supported as a top level element. Make sure you don't have other top level elements.")
 		}
 
-		for _, target := range element.FindElements("Target") {
-			locURI := target.FindElement("LocURI")
+		for _, locURI := range element.FindElements("//Target/LocURI") {
```

Comment:
Just curious, was that missing cases or is it just more efficient/cleaner way to achieve the same results?

Reply:
both! it looked more cleaner and turns out the hierarchy is `Item > Target > LocURI` (we were missing the `Item` part)

in retrospective... should the query then be `//Item/Target/LocURI`? currently we might get false positives, but maybe that's alright?

I will adjust in a separate PR to not miss the CI checks and both Gabe's and yours approval

Reply:
Yeah, sounds good to me to add the more specific query and to do it in a different PR.